lib/core/src/lib/form/components/widgets/date-time/date-time.widget.ts

```diff
@@ -17,73 +17,90 @@
-
-    minDate: Moment;
-    maxDate: Moment;
+    minDate: string;
+    maxDate: string;
 
     private onDestroy$ = new Subject<boolean>();
 
     constructor(public formService: FormService,
-                private dateAdapter: DateAdapter<Moment>,
-                private userPreferencesService: UserPreferencesService) {
+                private dateAdapter: DateAdapter<DateFnsAdapter>,
+                private userPreferencesService: UserPreferencesService,
+                @Inject(MAT_DATE_FORMATS) private dateFormatConfig: MatDateFormats,
+                private translationService: TranslationService) {
         super(formService);
     }
 
     ngOnInit() {
         this.userPreferencesService
             .select(UserPreferenceValues.Locale)
             .pipe(takeUntil(this.onDestroy$))
-            .subscribe(locale => this.dateAdapter.setLocale(locale));
+            .subscribe((locale) => this.dateAdapter.setLocale(DateFnsUtils.getLocaleFromString(locale)));
 
-        const momentDateAdapter = this.dateAdapter as MomentDateAdapter;
-        momentDateAdapter.overrideDisplayFormat = this.field.dateDisplayFormat;
+        this.dateFormatConfig.display.dateInput = this.field.dateDisplayFormat;
 
         if (this.field) {
             if (this.field.minValue) {
-                this.minDate = moment.utc(this.field.minValue, 'YYYY-MM-DDTHH:mm:ssZ');
+                this.parseDateAndSetMinMaxValue(this.field.minValue, 'minDate');
             }
 
             if (this.field.maxValue) {
-                this.maxDate = moment.utc(this.field.maxValue, 'YYYY-MM-DDTHH:mm:ssZ');
+                this.parseDateAndSetMinMaxValue(this.field.maxValue, 'maxDate');
             }
         }
     }
 
+    parseDateAndSetMinMaxValue = (dateString, targetProperty) => {
+        if (dateString && !dateString.includes('00.')) {
+            this.field[targetProperty] = DateFnsUtils.addSeconds(dateString);
+        }
+
+        const [year, month, day, hours, minutes, seconds] = dateString.split(/[-T:.Z]/).map(Number);
+        const parsedDate = new Date(Date.UTC(year, month - 1, day, hours, minutes, seconds));
+
+        if (isValid(parsedDate)) {
+            this[targetProperty] = parsedDate.toISOString();
+        }
+    };
+
     ngOnDestroy() {
         this.onDestroy$.next(true);
         this.onDestroy$.complete();
     }
 
+    formatDateTimeLabel(field: FormFieldModel): string {
+        const dateTimeDisplayName = this.translationService.instant(field.name);
+        const dateTimeDisplayFormat = DateFnsUtils.convertDateFnsToMomentFormat(field.dateDisplayFormat);
+
+        return `${dateTimeDisplayName} (${dateTimeDisplayFormat})`;
+    }
+
     onDateChanged(newDateValue) {
-        const date = moment(newDateValue, this.field.dateDisplayFormat, true);
-        if (date.isValid()) {
-            this.field.value = moment(date).utc().local().format(this.field.dateDisplayFormat);
```

Comment:
you are removing the "utc().local()" here, what is the "local" equivalent date/time is in the right side?

Reply:
I think the utc().local() conversion from moment.js is not needed in date-fns, as JavaScript's Date object automatically handles local time and UTC conversion based on the system's time zone settings.